src/Error/Debugger.php

```diff
@@ -918,10 +918,27 @@ public static function addFormat(string $format, array $strings): array
         } else {
             $self->_templates[$format] = $strings;
         }
+        unset($self->renderers[$format]);
 
         return $self->_templates[$format];
     }
 
+    /**
+     * Add a renderer to the current instance.
+     *
+     * @param string $name The alias for the the renderer.
+     * @param string $class The classname of the renderer to use.
+     * @return void
+     */
+    public static function addRenderer(string $name, string $class): void
```

Comment:
What does adding a renderer with an alias do? Would you have multiple html renderers?

Reply:
It was primarily so I could test out the override behavior of template updates to ensure backwards compatibility. I preferred this to a read-accessor for templates as I would like to remove templates and having more methods around them means more breaks in the future. With the addition of this method we're potentially unlocking migration to the new system and this method isn't one I would feel bad about keeping for another major revision.